File: src/ts/magma/app/compile/text/Symbols.ts
import { StripRule } from "../../../../magma/app/compile/rule/StripRule";
import { SymbolRule } from "../../../../magma/app/compile/rule/SymbolRule";
import { TypeRule } from "../../../../magma/app/compile/rule/TypeRule";
import { StringRule } from "../../../../magma/app/compile/rule/StringRule";
export class Symbols {
	static createSymbolRule(): StripRule {
		return new StripRule(new SymbolRule(new TypeRule("symbol", new StringRule("value"))))/*unknown*/;
	}
}

File: src/java/magma/app/compile/text/Symbols.java
package magma.app.compile.text;

import magma.app.compile.rule.StringRule;
import magma.app.compile.rule.StripRule;
import magma.app.compile.rule.SymbolRule;
import magma.app.compile.rule.TypeRule;

public class Symbols {
    public static StripRule createSymbolRule() {
        return new StripRule(new SymbolRule(new TypeRule("symbol", new StringRule("value"))));
    }
}
